week-1/K-functions-parameters/exercise4.js

```diff
@@ -1,9 +1,13 @@
 // Declare your function first - call it "add"
-
+function add() {
```

Comment:
Here, you want to create an add function that has parameters. So `function add() {` should be `function add(num1, num2) {`